tests/search/regexp/regexp.rb

```diff
@@ -151,12 +151,15 @@ def test_regexp
     assert_hitcount(make_query(make_term("title", "test2")), 3);
     assert_hitcount(make_query(make_term("title", "test3")), 3);
     check_uncased
-    check_cased
-
-    # Verify that indexed fields will fallback to 'contains'.
-    assert_hitcount(make_query(make_term("single_index", "thisisafox")), 3);
-    assert_hitcount(make_query(make_regexp("single_index", "thisisafox")), 3);
-    assert_hitcount(make_query(make_regexp("single_index", "^thisisafox")), 0);
+    # Cased regexp doesn't work in streaming mode
+    check_cased unless is_streaming
```

Comment:
https://github.com/vespa-engine/vespa/pull/29932 should resolve cased matching for streaming. Consider holding off until that PR is merged, at which point this special case can be removed